PWGJE/EMCALJetTasks/PWGJEEMCALJetTasksLinkDef.h

```diff
@@ -327,6 +327,7 @@
 #pragma link C++ class PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalSoftDropData+;
 #pragma link C++ class PWGJE::EMCALJetTasks::AliAnalysisTaskPtEfficiencyJets+;
 #pragma link C++ class PWGJE::EMCALJetTasks::AliAnalysisTaskTracksInJet+;
+#pragma link C++ class PWGJE::EMCALJetTasks::AliAnalysisTaskEmcalJetProperties+;
```

Comment:
Since you don't define you task in a namespace, you shouldn't include the namespace here. ROOT may or may not manage to load the task, but either way, this line should be fixed

Reply:
Could you please help me with where I have to change? I followed "AliAnalysisTaskEA.cxx" task and I included it accordingly.

Reply:
In Filip's task, he puts the class in the namespace `PWGJE::EMCALJetTasks`. See here: 
https://github.com/alisw/AliPhysics/blob/b81c559a8a5f7b5b547ec1212e40c520da0d50af/PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskEA.h#L52-L54

Further, I guess he uses fastjett objects directly (not wrapped in ALICE code), so his task needs to be there. You didn't define you task in a namespace, and although it's better practice, you don't need to do so. Nor do you access fastjer objects directly in your task. So all you need to do is removed that line, put after this line https://github.com/alisw/AliPhysics/blob/b81c559a8a5f7b5b547ec1212e40c520da0d50af/PWGJE/EMCALJetTasks/PWGJEEMCALJetTasksLinkDef.h#L134 :

```
 #pragma link C++ class AliAnalysisTaskEmcalJetProperties+; 
```

Reply:
Thank you very much. I have included our task in line 135.

Reply:
After adding this in line no 135, there is some conflicts error showing here. I do not know how to solve it. I have attached a screenshot
<img width="771" alt="Screenshot 2022-07-12 at 9 18 47 PM" src="https://user-images.githubusercontent.com/107457495/178534963-2ff7a315-1083-4311-96f6-9322c9d53ac9.png">